codex/marketplace.md

```diff
@@ -0,0 +1,209 @@
+---
+title: CODEX-MARKETPLACE
+name: Codex Storage Marketplace
+status: raw
+tags: codex
+editor: Dmitriy <dryajov@status.im>
```

Comment:
I would list Mark here as well (I can't tag him in this repo, he should be invited - https://github.com/markspanbroek)

Reply:
@markspanbroek